<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openrtk_qtClass</class>
 <widget class="QMainWindow" name="openrtk_qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>openrtk_qt</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Windows(&amp;W)</string>
    </property>
    <addaction name="actionTest"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_satellite">
   <property name="styleSheet">
    <string notr="true">QDockWidget &gt; QWidget { 
border: 1px solid gray; 
}</string>
   </property>
   <property name="windowTitle">
    <string>satellite</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockContents_satellite">
    <layout class="QVBoxLayout" name="vertical_satellite">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_log">
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray; </string>
   </property>
   <property name="windowTitle">
    <string>log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockContents_log">
    <layout class="QVBoxLayout" name="vertical_log">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_client">
   <property name="styleSheet">
    <string notr="true">QDockWidget &gt; QWidget { 
border: 1px solid gray; 
}</string>
   </property>
   <property name="windowTitle">
    <string>client</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockContents_client">
    <layout class="QVBoxLayout" name="vertical_client">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="openrtk_qt.qrc"/>
 </resources>
 <connections/>
</ui>
